.github/workflows/dependency-review.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: dependency review
+
+on:
+  pull_request:
+    branches: [development/*, stabilization/*, hotfix/*]
```

Comment:
I think you can remove it here as well (for consistency)

Reply:
Done!